packages/runtime-dom/src/directives/vOn.ts

```diff
@@ -10,6 +10,19 @@ import { hyphenate, isArray } from '@vue/shared'
 const systemModifiers = ['ctrl', 'shift', 'alt', 'meta']
 
 type KeyedEvent = KeyboardEvent | MouseEvent | TouchEvent
+type ModifierGuardsKeys =
+  | 'stop'
+  | 'prevent'
+  | 'self'
+  | 'ctrl'
+  | 'shift'
+  | 'alt'
+  | 'meta'
+  | 'left'
+  | 'middle'
+  | 'right'
+  | 'exact'
+  | ''
```

Comment:
Why is empty string allowed here?

Reply:
It used to be for compatibility here, but now it has been modified.

https://github.com/vuejs/core/blob/c0c9432b64091fa15fd8619cfb06828735356a42/packages/dts-test/defineComponent.test-d.tsx#L1504